core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -362,4 +366,30 @@ protected XContentBuilder innerToXContent(XContentBuilder builder, Params params
         }
         return builder;
     }
+
+    @Override
+    public String getWriteableName() {
+        return SUGGESTION_NAME;
+    }
+
+    @Override
+    public void doWriteTo(StreamOutput out) throws IOException {
+        // TODO
```

Comment:
Shouldn't this have an implementation before going in - even if it's just some "not implemented exception" to catch coding errors until there's the real code?